### PR TITLE
Reduce redundant logging

### PR DESCRIPTION
**Pull request by tanyatukade**

### CODE REVIEW

File: pkg/hostagent/nodes.go [hunk 1]
```diff
@@ -189,7 +189,7 @@ func (agent *HostAgent) registerHostVeth() {
 				vmName = fmt.Sprintf("%s.%s", vmName, agent.vtepIP)
 				ep.Attributes["vm-name"] = vmName
 			}
-			agent.log.Infof("-- Adding %+v to registry", ep)
+			agent.log.Debugf("-- Adding %+v to registry", ep)
```
Comment (snaiksat) [inline pkg/hostagent/nodes.go:192]: leave as info

File: pkg/hostagent/pods.go [hunk 2]
```diff
@@ -682,7 +682,7 @@ func (agent *HostAgent) epChanged(epUuid *string, epMetaKey *string, epGroup *me
 			"id": *epMetaKey,
 			"ep": neweps,
 		}).Debug("Updated endpoints for pod")
-		logger.Infof("EP: %+v", neweps[0])
+		logger.Debugf("EP updated: %+v", neweps[0])
```
Comment (snaiksat) [inline pkg/hostagent/pods.go:685]: leave as info

File: pkg/hostagent/snats.go [hunk 3]
```diff
@@ -1140,7 +1146,7 @@ func (agent *HostAgent) handleObjectDeleteForSnat(obj interface{}) {
 	var podidlist []string
 	sync := false
 	for name, resources := range plcynames {
-		agent.log.Infof("Handle snatpolicy as object deleted: %s,  ObjectKey: %s", name, objKey)
+		agent.log.Debugf("Handle snatpolicy as object deleted: %s, ObjectKey: %s", name, objKey)
```
Comment (snaiksat) [inline pkg/hostagent/snats.go:1149]: leave as info

**Comment by snaiksat** [comment]
We missed:
https://github.com/noironetworks/aci-containers/blob/master/pkg/controller/services.go#L281 (and similar fabric path updates)

Also have we accounted for these messages like these?

time="2021-12-07T16:22:25Z" level=info msg="Adding missing snatpolicy entry in snatglobalinfo for node: .."
time="2021-12-07T16:22:25Z" level=info msg="Snatpolicy list according to snatglobalinfo: map[...]"
time="2021-12-07T16:22:25Z" level=info msg="Snatpolicy list according to nodeinfo: map[installerclusterdefault:true]"
time="2021-12-07T16:22:25Z" level=info msg="Nodeinfo and globalinfo out of sync for node: ..."
time="2021-12-07T16:22:25Z" level=info msg="Queuing nodeinfokey for globalinfo sync: aci-containers-system/..."

Perhaps we should only leave the following at info level:
time="2021-12-07T16:22:25Z" level=info msg="Adding missing snatpolicy entry in snatglobalinfo for node: ..."

File: pkg/controller/controller.go [hunk 4]
```diff
@@ -725,14 +725,14 @@ func (cont *AciController) snatGlobalInfoSync(stopCh <-chan struct{}, seconds ti
...
-				cont.log.Info("Snatpolicy list according to nodeinfo: ", policyNames)
+				cont.log.Debug("Adding missing entry in snatglobalinfo for node: ", nodeName)
+				cont.log.Debug("No snat policies found in snatglobalinfo")
+				cont.log.Debug("Snatpolicy list according to nodeinfo: ", policyNames)
 				marked = true
 			} else if len(policyNames) != len(expectedmap[nodeName]) {
-				cont.log.Info("Adding missing snatpolicy entry in snatglobalinfo for node: ", nodeName)
-				cont.log.Info("Snatpolicy list according to snatglobalinfo: ", expectedmap[nodeName])
-				cont.log.Info("Snatpolicy list according to nodeinfo: ", policyNames)
+				cont.log.Debug("Adding missing snatpolicy entry in snatglobalinfo for node: ", nodeName)
```
Comment (snaiksat) [inline pkg/controller/controller.go:733]: should we leave this at Info?

**Comment by tanyatukade** [comment]
> We missed: https://github.com/noironetworks/aci-containers/blob/master/pkg/controller/services.go#L281 (and similar fabric path updates)
> 
> Also have we accounted for these messages like these?
> 
> time="2021-12-07T16:22:25Z" level=info msg="Adding missing snatpolicy entry in snatglobalinfo for node: .." time="2021-12-07T16:22:25Z" level=info msg="Snatpolicy list according to snatglobalinfo: map[...]" time="2021-12-07T16:22:25Z" level=info msg="Snatpolicy list according to nodeinfo: map[installerclusterdefault:true]" time="2021-12-07T16:22:25Z" level=info msg="Nodeinfo and globalinfo out of sync for node: ..." time="2021-12-07T16:22:25Z" level=info msg="Queuing nodeinfokey for globalinfo sync: aci-containers-system/..."
> 
> Perhaps we should only leave the following at info level: time="2021-12-07T16:22:25Z" level=info msg="Adding missing snatpolicy entry in snatglobalinfo for node: ..."

These have been taken into account and updated accordingly.

File: pkg/controller/services.go [hunk 5]
```diff
@@ -278,7 +278,7 @@ func (cont *AciController) fabricPathForNode(name string) (string, bool) {
 	sz := len(cont.nodeOpflexDevice[name])
 	for _, device := range cont.nodeOpflexDevice[name] {
 		if device.GetAttrStr("state") == "connected" {
-			cont.fabricPathLogger(device.GetAttrStr("hostName"), device).Info("Processing fabric path for node ",
+			cont.fabricPathLogger(device.GetAttrStr("hostName"), device).Debug("Processing fabric path for node ",
```
Comment (snaiksat) [inline pkg/controller/services.go:281]: In this patch lets just restore this to Info and we can consider a change in a separate patch.

**Review by snaiksat — APPROVED** [review]

**Pull request merged**